src/app/files/file-explorer.js

```diff
@@ -516,14 +517,12 @@ fileExplorer.prototype.renderMenuItems = function () {
         `
       } else {
         return yo`
-        <span onclick=${(event) => { event.stopPropagation(); this[ action ]() }} class="newFile ${css.newFile}" title=${title}>
-          <i class=${icon}></i>
-        </span>
+        <span onclick=${(event) => { event.stopPropagation(); this[ action ]() }} class="${icon} ${css.newFile}" title=${title}></span>
```

Comment:
for the sake of browser test :) we should keep the class here `newFile`. will do the edit